src/main/java/uk/gov/hmcts/reform/lrdapi/repository/CourtTypeServiceAssocRepository.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.lrdapi.repository;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+import uk.gov.hmcts.reform.lrdapi.domain.CourtTypeServiceAssoc;
+
+@Repository
+public interface CourtTypeServiceAssocRepository extends JpaRepository<CourtTypeServiceAssoc, String> {
+
+    @Query(value = "SELECT * FROM court_type_service_assoc ctsa WHERE ctsa.service_code = :serviceCode",
+        nativeQuery = true)
```

Comment:
any benefit here using native query over hibernate?

Reply:
Using just the hibernate query gives this error: org.postgresql.util.PSQLException: The column name "Code" was not found in this ResultSet.

Reply:
I think you need to add below code to fix this
`@Column(name = "service_code")
    @NotNull
    private String serviceCode;

    @ManyToOne
    @JoinColumn(name = "service_code",referencedColumnName = "service_code", insertable = false, updatable = false,
        nullable = false)
    private Service service;`
    
    in CourtTypeServiceAssoc.java

Reply:
Thanks @SujeethKumarAri, added this and it's working without any issues.